dev/conductor/lib/repository.dart

```diff
@@ -766,3 +793,68 @@ class Checkouts {
   final ProcessManager processManager;
   final Stdio stdio;
 }
+
+class CiYaml {
```

Comment:
What is the benefit to having conductor in the framework versus Cocoon?

Reply:
This is preferable as enabled_branches is straight forward, but if for some reason in the future we need more advanced logic it should rely directly on the backend.

Reply:
1. the benefit for the conductor living in the framework is so that it's knowledge of the Flutter SDK is inherently versioned with the SDK. For example, it has a hard-coded list of binaries that live in the tool cache that should be codesigned: https://github.com/flutter/flutter/blob/master/dev/conductor/lib/codesign.dart#L137.
2. it wouldn't be too hard to hook up to the conductor the ability to run scripts from `cocoon/app_dart/bin`. However, if we started doing this, I'd probably want to start pinning a cocoon version, at the minimal for the benefit of the conductor. Or have something like a linearized flutter revision list.

Reply:
2. This is fine. If there's enough feature requests, we can refactor a more portable `package:ci_yaml` for us to pass around.

Reply:
Hopefully without any pub deps, as the framework transitive deps situation is already a pain.